go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ervitis/filecache
```

Comment:
This isn't right for this repo; it would work for your own fork.

Reply:
@kisom sorry! I will update it